app/components/menubar.js

```diff
@@ -27,11 +31,11 @@ export default function Menubar(props) {
   const [collapsed, setCollapsed] = useState(true);
   const userCookie = Cookies.get("user");
   const hasAllRequiredCreds =
-    process.env.NEXTAUTH_URL &&
-    process.env.ROCKETCHAT_CLIENT_ID &&
-    process.env.ROCKETCHAT_CLIENT_SECRET &&
-    process.env.ROCKETCHAT_URL;
+    process.env.NEXT_PUBLIC_GOOGLE_CLIENT_ID &&
+    process.env.NEXT_PUBLIC_RC_URL
+
```

Comment:
Great work, here! This is the start and eventually, we will remove the legacy RocketChat authentication based on simple Oauth + NextAuth.

Reply:
Hey!! I've made the requested changes😄